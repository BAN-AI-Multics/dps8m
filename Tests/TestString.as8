
start3:

hello3: lda msg3,   sc
        emcall      1
        ttf         hello3
        "tra         L0
	eax2	L0
	tra	out
L0:     dis
    
	
out:	lda 	msg4, sc
        emcall  1
        ttf    out
	tra	0,x2

"
" data ...
"
msg3:   tallyb      msg3d, 40, 0 
msg3d:  aci         /This is a test of C-Style escapes ...\r\n/
msg4:   tallyb      msg4d, 24, 0 
msg4d:  aci         /... and we're back ...\r\n/
