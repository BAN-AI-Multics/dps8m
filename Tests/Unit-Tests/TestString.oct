000000 xxxx 000012235052 hello3: lda msg3,   sc
000001 xxxx 000001420400         emcall      1
000002 xxxx 000000607000         ttf         hello3
000003 xxxx 000005622000 	eax2	L0
000004 xxxx 000006710000 	tra	out
000005 xxxx 000000616000 L0:     dis
000006 xxxx 000025235052 out:	lda 	msg4, sc
000007 xxxx 000001420400         emcall  1
000010 xxxx 000006607000         ttf    out
000011 xxxx 000000710012 	tra	0,x2
000012 xxxx 000013005040 msg3:   tallyb      msg3d, 40, 0  
000013 xxxx 124150151163 msg3d:  aci         /This is a test of C-Style escapes ...\r\n/ 
000014 xxxx 040151163040  
000015 xxxx 141040164145  
000016 xxxx 163164040157  
000017 xxxx 146040103055  
000020 xxxx 123164171154  
000021 xxxx 145040145163  
000022 xxxx 143141160145  
000023 xxxx 163040056056  
000024 xxxx 056015012000  
000025 xxxx 000026003040 msg4:   tallyb      msg4d, 24, 0  
000026 xxxx 056056056040 msg4d:  aci         /... and we're back ...\r\n/ 
000027 xxxx 141156144040  
000030 xxxx 167145047162  
000031 xxxx 145040142141  
000032 xxxx 143153040056  
000033 xxxx 056056015012  
