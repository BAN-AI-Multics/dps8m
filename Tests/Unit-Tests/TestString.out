
dps-8/m simulator V4.0-0 Beta        git commit id: f386776a
28 (000034) words loaded
This is a test of C-Style escapes ...
... and we're back ...

cpuCycles = 196
TestString-10> g 0

DIS instruction, IC: 000005 (000000616000)
Goodbye
